### PR TITLE
Databases week1

### PR DESCRIPTION
**Pull request by saloumeh-67**

### CODE REVIEW

File: databases/week1/datab.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ALTER TABLE
+	status CHANGE id id int unsigned NOT NULL;
```
Comment (Elia-Git) [inline databases/week1/datab.sql:2]: Unrelevant queries, probably by mistake. Remove them.

File: databases/week1/datab.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	select COUNT(*)from task;
+	--Find out how many tasks in the task table do not have a valid due date
+	SELECT * from task where due_date is NULL;
```
Comment (Elia-Git) [inline databases/week1/datab.sql:5]: The small modification. See if it makes more sense...
SELECT count(title) from task where due_date is NULL;

File: databases/week1/datab.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	--Find out how many tasks in the task table do not have a valid due date
+	SELECT * from task where due_date is NULL; 
+	--Find all the tasks that are marked as done.
+	SELECT * from task 
+    join status on task.status_id = status.id 
+    where status.name  = 'Done';
+	-- Find all the tasks that are not marked as done
+	SELECT * from task 
+    join status on task.status_id = status.id 
+    where status.id < 3;
```
Comment (Elia-Git) [inline databases/week1/datab.sql:13]: Try this way instead
```
SELECT * from task 
    join status on task.status_id = status.id 
    where status.name  != 'Done';
```

File: databases/week1/datab.sql [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    --Get all the tasks, sorted with the most recently created first
+	SELECT MAX(created) from task GROUP BY user_id;
```
Comment (Elia-Git) [inline databases/week1/datab.sql:15]: Close to get it.
Try this way and you don't need to use the GROUP BY Clause. but ORDER BY
`SELECT title, created from task ORDER BY created DESC;`

File: databases/week1/datab.sql [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+	--Get the single most recently created task
+	SELECT MAX(created)from task;
```
Comment (Elia-Git) [inline databases/week1/datab.sql:17]: Same here 
`SELECT title, created from task ORDER BY created DESC LIMIT 1;`

File: databases/week1/datab.sql [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+	--Get the title and due date of all tasks where the title or description contains database
+	SELECT title, due_date FROM task WHERE title LIKE "%database%" OR description LIKE "%database%";
+	--Get the title and status (as text) of all tasks
+	SELECT task.title, status.name from task,status where task.status_id = status_id;
```
Comment (Elia-Git) [inline databases/week1/datab.sql:21]: A small typo fix
`SELECT task.title, status.name from task,status where task.status_id = status.id;`

File: databases/week1/datab.sql [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+	--Get the name of each status, along with a count of how many tasks have that status
+    SELECT status.name, count(*) FROM status JOIN task ON task.status_id = status.id
+    GROUP BY status.name; 
+	--Get the names of all statuses, sorted by the status with most tasks first
+   SELECT status.name, count(*) FROM status JOIN task ON task.status_id = status.id
+   GROUP BY status.name 
+   ORDER BY count(*) DESC; 
+   --I wanted to use orderby but I coulden't find out how it would be nice if you help with that :).
```
Comment (Elia-Git) [inline databases/week1/datab.sql:29]: Good job here!
Do you still need support about ORDER BY concepts?  Read some cool pieces of stuff here whehttps://www.w3schools.com/sql/sql_orderby.asp

**Review by Elia-Git — COMMENTED** [review]
Keep fighting!
Would like to see you implement the given feedback before merging. 

Thnanks!

File: databases/week1/datab.sql (hunk 3 above)
Comment (saloumeh-67) [reply databases/week1/datab.sql:13]: smart:)

**Review by Elia-Git — COMMENTED** [review]
Great to see your implementation, thanks!
Make sure always to make your code look nice by doing all necessary indentation.
For instance you can read something about indentation in MySQL here and make changes if you feel comfortable,, https://courses.cs.washington.edu/courses/cse154/18au/resources/styleguide/sql/spacing-indentation-sql.html
And then you can ship it.

**Comment by saloumeh-67** [comment]
I will thanks a lot 🥰🌺

**Pull request merged**